src/module.api/poolpair.service.ts

```diff
@@ -425,3 +429,194 @@ function findPoolSwapFromTo (history: AccountHistory, from: boolean): PoolSwapFr
 
   return undefined
 }
+
+@Injectable()
+export class PoolSwapPathFindingService {
+  tokenGraph: UndirectedGraph = new UndirectedGraph()
+
+  constructor (
+    protected readonly poolPairTokenMapper: PoolPairTokenMapper,
+    protected readonly deFiDCache: DeFiDCache
+  ) {
+  }
+
+  @Interval(10_000) // 10s
```

Comment:
```suggestion
  @Interval(120_000) // 10s
```

In the real world, the PoolPair is mostly constant, maybe we can change it to a larger number? 🤔